### PR TITLE
Pets index styling

### PR DESCRIPTION
**Pull request by jadekstewart3**
### Describe your change in plain English.
Update "/org/pets" to utilize the geeksui styling.
Move the "Our Pets" to the header section
Move the "Create Pet " link to the upper right and hand corner and make it a button instead of a link

### Link to the issue
#190

### CODE REVIEW

File: app/views/organizations/pets/index.html.erb [hunk 1]
```diff
@@ -14,12 +14,21 @@
...
-        <%= t('.our_pets') %>
-      </h1>
-    </div>
+    <%= content_for :header do  %>
+      <div class="text-center mb-5">
+        <h2 class="section-heading text-uppercase">
+          <%= t('.our_pets') %>
+        </h2>
+      </div>
+    <% end %>
```
Comment (edwinthinks) [inline app/views/organizations/pets/index.html.erb:23]: I think we should reduce the number of details in the header to be a bit smaller. This is because you'll have to repeat the div & h2 classes for every page. Instead could we do something like:
```
<%= provide(:header_title, t(".our_pets")) %>
```
and in the layout have something like:
```
      <div class="text-center mb-5">
        <h2 class="section-heading text-uppercase">
          <%= yield(:header_title) %>
        </h2>
      </div>
```

This is going to make the amount of information each page needs to write less. Let me know what you think :)

File: app/views/organizations/pets/index.html.erb [hunk 2]
```diff
@@ -14,12 +14,21 @@
...
 
+    <!--create pet button-->
+      <%= content_for :button do  %>
+        <div class ="col-md-6 d-flex justify-content-center align-items-center">
+          <%= button_to t('.create_pet'), new_pet_path, class: "btn btn-primary" %><br>
+        </div>
```
Comment (edwinthinks) [inline app/views/organizations/pets/index.html.erb:29]:
```suggestion
          <%= button_to t('.create_pet'), new_pet_path, class: "btn btn-primary" %><br>
```
I think you can change it to this since the `col-md-6` is shrinking the button and I think it will look fine without the surrounding `div`

**Review by edwinthinks — CHANGES_REQUESTED** [review]
Nice work @jadekstewart3 ! A few things to move around but it looks good! Let me know if you want to clarify on any of my comments.

File: app/views/organizations/pets/index.html.erb [hunk 3]
```diff
@@ -14,12 +14,21 @@
   <div class="container mb-md-5 mt-md-5">
 
     <!--page heading-->
-    <div class="text-center mb-5">
-      <h1 class="section-heading text-uppercase underline">
-        <%= t('.our_pets') %>
-      </h1>
-    </div>
+    <%= content_for :header do  %>
```
[the remark this answers is not in the document]
Comment (jadekstewart3) [reply app/views/organizations/pets/index.html.erb:17]: @edwinthinks Thank you! I'm going to get on this right now!

**Review by edwinthinks — APPROVED** [review]
After you've fixed the merge conflict..... LGTM!

Thanks for tackling this!

**Comment by edwinthinks** [comment]
LGTM!
